dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -4446,6 +4446,14 @@ Sedp::write_subscription_data_unsecure(
                  ACE_TEXT("to ParameterList\n")));
       result = DDS::RETCODE_ERROR;
     }
+
+    DDS::TypeConsistencyEnforcementQosPolicy tceqp = TheServiceParticipant->initial_TypeConsistencyEnforcementQosPolicy();
```

Comment:
@sonndinh @ClaytonCalabrese @mitza-oci Sending `ignore_member_names` set to `false`.